Tests/SwiftProtobufTests/Test_Struct.swift

```diff
@@ -288,18 +281,6 @@ class Test_JSON_Value: XCTestCase, PBTestHelpers {
     }
 
     func testValue_list() throws {
-        XCTAssertEqual(Google_Protobuf_ListValue(any: [1]),
```

Comment:
Just to make sure I understand:  It looks like you dropped the initializers that take `Swift.Any`, so you've also dropped these corresponding tests?